Tasks/Common/webdeployment-common/deployusingmsdeploy.ts

```diff
@@ -93,7 +142,15 @@ async function executeMSDeploy(msDeployCmdArgs) {
     });
 
     try {
-        await tl.exec("msdeploy", msDeployCmdArgs, <any>{failOnStdErr: true, errStream: errObj});
+        tl.debug("the argument string is:");
+        tl.debug(msDeployCmdArgs);
```

Comment:
Woah - why do you need to mask?  If you register secrets with the agent then the agent masks for you.  We do not want duplicate masking logic.